subsys/net/lib/downloader/src/shell.c

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/device.h>
+#include <stdlib.h>
+#include <string.h>
+#include <zephyr/shell/shell.h>
+#include <net/downloader.h>
+
+LOG_MODULE_DECLARE(downloader);
+
+static int dl_callback(const struct downloader_evt *event);
+
+static char dl_buf[2048];
```

Comment:
Comment on the 2048 here too like in `transports/http.c`?

Reply:
It is not the same.